doc/conf.py

```diff
@@ -184,8 +184,9 @@
 docscrape.ClassDoc.extra_public_methods = mne.utils._doc_special_members
 numpydoc_class_members_toctree = False
 numpydoc_show_inherited_class_members = {
-    "mne.SourceSpaces": False,
     "mne.Forward": False,
+    "mne.Projection": False,
```

Comment:
I don't think it makes sense to document the `dict` methods for a `Projection`. Users are not suppose to do `Projection.update(...)` anyway.